site/en/blog/memory-and-energy-saver-mode/index.md

```diff
@@ -0,0 +1,177 @@
+---
+layout: 'layouts/blog-post.njk'
+title: What developer need to know about Chrome's Memory and Energy Saver modes
+date: 2022-12-08
+authors:
+  - philipwalton
+description: Learn how to ensure your page can gracefully handle Chrome's new Memory and Energy saver modes.
+hero: image/eqprBhZUGfb8WYnumQ9ljAxRrA72/J07wJiRCtOoE1sleUViq.png
+alt: Graphical icons representing Chrome's Memory and Energy Saver modes
+tags:
+  - performance
+  - chrome-108
+---
+
+Chrome 108 [introduced](https://blog.google/products/chrome/new-chrome-features-to-save-battery-and-make-browsing-smoother/) two new modes, [Memory Saver and Energy Saver](https://support.google.com/chrome/answer/12929150), to give users more control over how Chrome utilizes their system resources.
+
+While these new modes are primarily user-facing, they do have some implications that are important for web developers to be aware of, as they can potentially impact the user experience of your site.
+
+This post will cover the potential effects of these new modes and what web developers can do to ensure they're delivering the best experience possible.
+
+## Memory Saver mode
+
+When Memory saver mode is enabled, Chrome will proactively discard tabs that have been unused in the background for some time. This frees up memory for active tabs as well as other applications that may be running. Users can instruct Chrome not to discard tabs for specific sites; however, this is a user preference and not something you can control as a developer.
+
+When a tab is discarded, its title and favicon still appear in the tab strip but the page itself is gone, exactly as if the tab had been closed normally. If the user revisits that tab, the page will be reloaded automatically.
+
+For purely content pages, discarding and reloading a tab will likely not affect the user experience, but for rich, interactive sites with complex user flows, a reload in the middle of that flow could be extremely frustrating if the site is not able to restore the page to exactly where the user left off.
+
+Discarding tabs to conserve memory is something Chrome has been doing for years, but it was only done in situations where the system was under critical memory pressure. Given its relatively rare occurrence, web developers may not have realized it was happening.
```

Comment:
I think we can come back to this (if it's a sticking point), but I did want to be a bit vague here since we aren't sharing any specific data about how rare it was.

The point I want to emphasize is that the behavior isn't changing, it's just going to become more common.